package.json

```diff
@@ -20,8 +20,8 @@
   "main": "index.js",
   "files": [
     "bin/",
-    "src/",
-    "index.js"
+    "index.js",
```

Comment:
Changes other than added `yargs` and `lodash.defaults` is npm sorting keys and strings